go.mod

```diff
@@ -1,3 +0,0 @@
-module github.com/datumbrain/jcsv
-
-go 1.16
```

Comment:
Why `go.mod` is deleted?

Reply:
go was not able to compile the code.
I removed it from this directory and moved it into the parent directory.
I then changed the 1st line to module [ParentFolderName].
package main was also in that folder.
Then the code compiled fine.

Reply:
`go.mod` should be in this package, and it should compile. There should be no `main` package here. You can create a `main` package inside this package for testing, but don't commit it.